src/main.cpp

```diff
@@ -6019,10 +6024,9 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
     else if (strCommand == NetMsgType::BLOCK && !fImporting && !fReindex) // Ignore blocks received while importing
     {
         CBlock block;
+        pfrom->nBlockSize = vRecv.size(); // Used by ProcessNewBlock() debug
```

Comment:
@sipa thanks. ok, so I need to add the size of the header to get the block size, or subtract it? Do you say it's not needed as there is the function GetSerializeSize()? Happy to call that instead, but I somehow felt I was saving resources by not needing to.